devtools/zenodo/zenodo_data_release.py

```diff
@@ -87,24 +87,26 @@ def __init__(self, env: str):
 
         logger.info(f"Using Zenodo token: {token[:4]}...{token[-4:]}")
 
-    def retry_request(self, *, method, url, max_tries=5, timeout=5, **kwargs):
+    def retry_request(self, *, method, url, max_tries=6, timeout=2, **kwargs):
```

Comment:
I changed the function to do an exponential backoff starting at 2 seconds, up to 64 seconds by default (previously it started at 5 seconds, and then went to 2 seconds for the 2nd try).